crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -176,6 +176,27 @@ def create_proposal
     base
   end
 
+  def disable_non_core_proposals
+    # Find all non-core proposals and remove all roles that belong
+    # to those proposals from the nodes
+    active_non_core_roles = RoleObject.find_roles_by_name("*-config-*").reject { |r| r.core_role? }
```

Comment:
Style/SymbolProc: Pass `&:core_role?` as an argument to `reject` instead of a block.